website/docs/r/lambda_event_source_mapping.html.markdown

```diff
@@ -69,8 +69,16 @@ resource "aws_lambda_event_source_mapping" "example" {
 
 ## Import
 
-Lambda Event Source Mappings can be imported using the `UUID` (event source mapping identifier), e.g.
+Lambda event source mappings can be imported using the `UUID` (event source mapping identifier), e.g.
 
 ```
 $ terraform import aws_lambda_event_source_mapping.event_source_mapping 12345kxodurf3443
 ```
+
+Note: AWS does not expose `startingPosition` when getting Lambda event source mappings, as such, if
```

Comment:
Thanks for documenting this behaviour. I don't think we want to recommend that practitioners modify the state file directly. Seeing as recreating the mapping is a necessary step given the API limitation providing information on why the recreate is happening should be sufficient.

Reply:
So removing the `, unless ....` is what you mean?

Reply:
Yes, I recommend that you use the `~> **Note**` to create an info box.

```suggestion
~> **Note:** Terraform will recreate the imported Lambda event source mappings as AWS does not expose `startingPosition` information when getting Lambda event source mappings.

For information about retrieving event source mappings, see [GetEventSourceMapping][3] in the API docs.

[3]: https://docs.aws.amazon.com/lambda/latest/dg/API_GetEventSourceMapping.html
```

Reply:
Updated -- will that add the bit below into the info box as well? Are there docs on how the website builder is done?